src/EFCore.SqlServer/Extensions/SqlServerDbFunctionsExtensions.cs

```diff
@@ -16,6 +16,44 @@ namespace Microsoft.EntityFrameworkCore;
 /// </remarks>
 public static class SqlServerDbFunctionsExtensions
 {
+    /// <summary>
+    ///     Extracts a scalar value from a JSON string. <paramref name="propertyReference" /> and <paramref name="path" />.
+    ///     Corresponds to SQL Server's <c>JSON_VALUE(@propertyReference, @path)</c>.
+    /// </summary>
+    /// <remarks>
+    ///     See <see href="https://aka.ms/efcore-docs-database-functions">Database functions</see>, and
+    ///     <see href="https://aka.ms/efcore-docs-sqlserver">Accessing SQL Server and SQL Azure databases with EF Core</see>
+    ///     for more information and examples.
+    /// </remarks>
+    /// <param name="_">The <see cref="DbFunctions" /> instance.</param>
+    /// <param name="propertyReference">The property in table</param>
+    /// <param name="path">The json path.</param>
+    /// <returns>Value of path</returns>
+    public static string JsonValue(
+        this DbFunctions _,
+        object propertyReference,
```

Comment:
JSON_VALUE doesn't have to accept a column, it can be any string expression that evaluates to a JSON string (here again, am following the SQL Server docs).

```suggestion
        object expression,
```